run_tests.sh

```diff
@@ -178,6 +183,9 @@ elif [[ "$1" == 'test' ]]; then
   files_list=$(find "$TESTS_DIR" | grep --perl-regexp "${regex}" | grep --extended-regexp --invert-match 'samples/.*|/shunit2/')
   set_tests $files_list
   LANGUAGE=en_US.UTF_8 run_tests
+elif [[ "$1" == "--clear-cache" ]]; then
+  say "Clearing tests cache"
```

Comment:
Use single quotes; also, add this option in the help output.

Reply:
ok